src/Language/PureScript/Docs/Prim.hs

```diff
@@ -462,6 +463,15 @@ quoteDoc = primTypeOf (P.primSubName "TypeError") "Quote" $ T.unlines
   , "[the Custom Type Errors guide](https://github.com/purescript/documentation/blob/master/guides/Custom-Type-Errors.md)."
   ]
 
+quoteLabelDoc :: Declaration
+quoteLabelDoc = primTypeOf (P.primSubName "TypeError") "QuoteLabel" $ T.unlines
+  [ "The `QuoteLabel` type constructor will produce a `Doc` when given a `Symbol`. When the resulting `Doc` is rendered "
+  , "for a `Warn` or `Fail` constraint, a syntactically valid label will be produces, escaping with quotes as needed."
```

Comment:
Typo: `will be produces` -> `will be produced`

Reply:
ugh, I even read it 4 times :/ sorry